DBQnA/ui/docker/Dockerfile.react

```diff
@@ -16,6 +21,10 @@ RUN ["npm", "run", "build"]
 
 FROM nginx:alpine
 
+ARG texttosql_url
+
+ENV TEXT_TO_SQL_URL=$texttosql_url
```

Comment:
Why not pass this when we start the container? CICD won't pass any ENV while building the release images.

Reply:
For now all the ENVs pass during runtime.

Reply:
But the docker image is common used not for a special ${ip_address}, if you pass the value during build, I am afraid the released image could not be widely used in different machine.